modules/sitemaps/sitemap-constants.php

```diff
@@ -7,6 +7,16 @@
  * @author Automattic
  */
 
+/**
+ * Number of seconds between sitemap and news updates in development code.
+ *
+ * @since 5.x.x
+ */
+if ( defined( 'JETPACK_DEV_DEBUG' ) && JETPACK_DEV_DEBUG ) {
+	define( 'JP_SITEMAP_INTERVAL', 60 );
```

Comment:
Do you think we should check if the constants are already defined here as well, before to overwrite them, just in case someone wants to overwrite that overwrite?